Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -1522,6 +1477,64 @@ namespace OpenInfraPlatform {
 				return paths;
 			}
 
+			// Function 10: Get rectangle coordiantes
+			
+			void GetRectangleCoordinates (std::vector<carve::geom::vector<2>>& coords, double radius, double x, double y) const

```

Comment:
It is best, if input parameters are marked as const:
```suggestion
			void GetRectangleCoordinates (std::vector<carve::geom::vector<2>>& coords, const double radius, const double x, const double y) const

```